packages/react/tests/e2e-storybook/cypress/integration/Footer/Footer.e2e.js

```diff
@@ -165,17 +168,19 @@ describe('Footer | Short (desktop)', () => {
 
     cy.wait(500);
 
-    cy.get('.bx--locale-modal [data-region="mea"]')
+    cy.get(
+      '[data-autoid="dds--locale-modal"] [data-autoid="dds--card"][data-region="mea"]'
+    )
       .find('a')
       .click();
 
     cy.wait(500);
 
-    cy.get('.bx--locale-modal__filter')
-      .find('.bx--locale-modal__list li a')
+    cy.get('[data-autoid="dds--locale-modal"]')
+      .find('ul li a[href][data-region]')
       .each($locale => {
         if (!$locale.attr('data-region') === 'mea') {
-          $locale.should('have.class', 'bx--locale-modal__locales-hidden');
+          $locale.should('have.css', 'display', 'none');
```

Comment:
Getting others' opinions, wondering if the assertion should be this instead:

```$locale.should('not.be.visible');```

cc: @annawen1 @ariellalgilmore

Reply:
yeah I like that instead of checking the css!

Reply:
agree! probably want to avoid relying on css when possible

Reply:
Thanks! I updated the assertion to not rely on css.